### ISSUE TITLE
# in user bio breaks settings export

### ISSUE THREAD

**Issue by Asudox** [LemmyNet/lemmy#5004]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [X] Do you agree to follow the rules in our [Code of Conduct](https://join-lemmy.org/docs/code_of_conduct.html)?
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Summary

Basically, if a # is anywhere in the user bio, the settings export breaks. The JSON is cut at the # and becomes unable to be imported.

### Steps to Reproduce

1. Go to account settings
2. Add a # anywhere in the bio
3. Export the settings

### Technical Details

No errors while exporting

### Version

BE: 0.19.3

### Lemmy Instance URL

programming.dev

**Comment by Nutomic** [comment]
Duplicate of https://github.com/LemmyNet/lemmy-ui/issues/2413
